solr/core/src/java/org/apache/solr/analysis/TokenizerChain.java

```diff
@@ -49,7 +49,6 @@ public TokenizerChain(CustomAnalyzer customAnalyzer) {
         customAnalyzer.getTokenizerFactory(),
         customAnalyzer.getTokenFilterFactories().toArray(new TokenFilterFactory[0]));
     setPositionIncrementGap(customAnalyzer.getPositionIncrementGap(null));
-    setVersion(customAnalyzer.getVersion());
```

Comment:
It surprises. me that there is no replacement here.

Reply:
Yeah, initially I left it as a nocommit figuring it must have moved or something, but root issue essentially says, this has not been used in a long time, didn't work very well when it was used, just removing it.

Reply:
I see that you removed the call to setVersion but otherwise didn't touch a thing.  Maybe we should output a warning if you set the version (no longer supported)?

Reply:
Also, anyone coming along to read this code (as I just was for something unrelated) finds it a bit curious to read the version only to do nothing with it.  So even if we don't agree on a warning perhaps, surely a comment is warranted.